react-exercise/src/types/filter.ts

```diff
@@ -0,0 +1 @@
+export type FilterType = "Cities" | "Age" | null;
```

Comment:
We usually use optional arguments with undefined value (no really good reason I think, mainly a convention) - so no need to add "null" to the type itself

Reply:
You mean that I should do something like this ?
export type FilterType = "Cities" | "Age";

const [activeFilter, setActiveFilter] = useState<FilterType | undefined>(undefined);